package.json

```diff
@@ -45,8 +45,18 @@
     "type": "git",
     "url": "git://github.com/ramda/ramda.git"
   },
-  "module": "es/index.js",
-  "main": "src/index",
+  "main": "./src/index.js",
+  "exports": {
+    ".": {
+      "require": "./src/index.js",
+      "import": "./es/index.js",
+      "default": "./src/index.js"
```

Comment:
`"require"` and `"default"` are serving the same purpose - CJS consumers, problem is that there was an experimental version of node@13 which had support for `"import"` but had no notion of the `"require"` key, so this "duplication" handles both versions at the same time as that experimental version of node can just fallback to the `"default"` whereas the new version will match against `"require"`